test/suites/backup.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+test_container_import() {
+  ensure_import_testimage
+
+  lxc_remote init testimage ctImport
+  lxc_remote snapshot ctImport
+  lxc start ctImport
+  ! lxd import ctImport
+  lxd import ctImport --force
+  lxc_remote info ctImport | grep snap0
+  lxc_remote delete --force ctImport
+ 
+  lxc_remote init testimage ctImport
+  lxc_remote snapshot ctImport
+  lxc start ctImport
+  sqlite3 "${LXD_DIR}/lxd.db" "DELETE FROM containers WHERE name='ctImport'"
+  ! lxd import ctImport
+  lxd import ctImport --force
+  lxc_remote info ctImport | grep snap0
+  lxc_remote delete --force ctImport
+
+  lxc_remote init testimage ctImport
+  lxc_remote snapshot ctImport
+  lxc start ctImport
+  sqlite3 "${LXD_DIR}/lxd.db" "DELETE FROM containers WHERE name='ctImport/snap0'"
+  ! lxd import ctImport
+  lxd import ctImport --force
+  lxc_remote info ctImport | grep snap0
+  lxc_remote delete --force ctImport
+
+  lxc_remote init testimage ctImport
+  lxc_remote snapshot ctImport
+  lxc start ctImport
+  sqlite3 "${LXD_DIR}/lxd.db" "DELETE FROM containers WHERE name='ctImport'"
+  sqlite3 "${LXD_DIR}/lxd.db" "DELETE FROM containers WHERE name='ctImport/snap0'"
+  sqlite3 "${LXD_DIR}/lxd.db" "DELETE FROM storage_volumes WHERE name='ctImport'"
+  ! lxd import ctImport
+  lxd import ctImport --force
+  lxc_remote info ctImport | grep snap0
+  lxc_remote delete --force ctImport
+
+  lxc_remote init testimage ctImport
+  lxc_remote snapshot ctImport
+  lxc start ctImport
+  sqlite3 "${LXD_DIR}/lxd.db" "DELETE FROM containers WHERE name='ctImport'"
+  sqlite3 "${LXD_DIR}/lxd.db" "DELETE FROM containers WHERE name='ctImport/snap0'"
+  sqlite3 "${LXD_DIR}/lxd.db" "DELETE FROM storage_volumes WHERE name='ctImport'"
+  sqlite3 "${LXD_DIR}/lxd.db" "DELETE FROM storage_volumes WHERE name='ctImport/snap0'"
+  lxd import ctImport
+  lxd import ctImport --force
+  lxc_remote info ctImport | grep snap0
+  lxc_remote delete --force ctImport
```

Comment:
"lxc_remote" is only for when you're talking to another LXD where a certificate is needed. Everything above seems to be done on a single LXD instance so there shouldn't be any needed for TLS communication.